rafs/src/metadata/direct_v6.rs

```diff
@@ -356,6 +356,29 @@ impl RafsSuperInodes for DirectSuperBlockV6 {
             inode.name();
             return Ok(Arc::new(inode));
         }
+
+        let root_ino = self.root_ino();
+        let root = self.get_inode(root_ino, false)?;
+        let mut stack = VecDeque::new();
+        if root.is_dir() {
+            stack.push_back(root);
+        }
+        while !stack.is_empty() {
+            let parent = stack.pop_front().unwrap();
+            for child_idx in 0..parent.get_child_count() {
+                if let Ok(child) = parent.get_child_by_index(child_idx) {
+                    if child.ino() == inode.ino() {
+                        inode.parent_inode.set(Some(parent.ino()));
+                        return Ok(Arc::new(inode));
+                    }
+
+                    if child.is_dir() {
+                        let child_inode = self.get_inode(child.ino(), false)?;
+                        stack.push_back(child_inode);
+                    }
+                }
+            }
+        }
```

Comment:
We'd better make this optional in order not to affect nydusd runtime

Reply:
Fixed.